index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
```

Comment:
The project is not in presentable form and is broken. Kindly see the screenshot below and make it exact same as in the design.
![broken](https://user-images.githubusercontent.com/35504149/138149622-d6e89796-94f4-4390-900e-cd566640da8a.png)